Assets/Python/pyWB/CvWBDesc.py

```diff
@@ -1907,8 +1907,10 @@ def read(self, fileName):
 		return 0
 
 	# after reading setup the map
-	def applyMap(self):
-		self.gameDesc.apply()
+	def applyMap(self, shouldApplyYearTurn = True):
+		if shouldApplyYearTurn:
```

Comment:
Pretty good. shouldApplyYearTurn could be GAME.getCurrentMap() == MapTypes.MAP_EARTH: